server/src/test/scala/services/crunch/CrunchSplitsToLoadAndDeskRecsSpec.scala

```diff
@@ -156,14 +164,17 @@ class CrunchSplitsToLoadAndDeskRecsSpec extends CrunchTestLike {
             SplitRatio(eeaMachineReadableToDesk, 0.25)
           )))
 
-        crunch.liveArrivalsInput.offer(flights)
-
-        val result = getLastMessageReceivedBy(crunch.liveTestProbe, 3 seconds)
-        val resultSummary = paxLoadsFromPortState(result, 5)
+        offerAndWait(crunch.liveArrivalsInput, flights)
 
         val expected = Map("T1" -> Map(Queues.EeaDesk -> Seq(5.0, 0.0, 0.0, 0.0, 0.0)))
 
-        resultSummary === expected
+        crunch.liveTestProbe.fishForMessage(30 seconds) {
```

Comment:
presumably we can make these timeouts a bit lower - although I guess it will only matter if the test fails.